parquet_fe_prototype/search.py

```diff
@@ -20,12 +38,17 @@ def initialize_index(datapackage: Package) -> index:
     """
     storage = RamStorage()
 
-    analyzer = RegexTokenizer(r"[^\s_]+") | LowercaseFilter() | StopFilter()
+    analyzer = (
```

Comment:
an analyzer is a little pipeline, starting with a tokenizer (turns a big long string into a generator of individual words/"tokens"). Then the stream of tokens gets passed through all these different filters, which operate on the individual tokens themselves.

* `LowercaseFilter` makes everything lowercase.
* `StopFilter` removes common "stopwords" like `it`, `is`, `a`, etc.
* `StemFilter` takes a stemmer function, and trims words to their "stems" to allow for some amount of fuzzy matching.